# Standard ROS header
Header header

# Array of angles
float32[] angles

# Number of joints
uint32 count
